_common.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euxfo pipefail;
+
+USERNAME="${DOCKER_USERNAME:-jackfirth}"
```

Comment:
Does the `:-jackfirth` part make this default to `jackfirth` if `DOCKER_USERNAME` isn't defined?

I lean towards making this variable required with no default or not making it configurable at all. IMO, the fewer configuration knobs a shell script has the better.

Reply:
> Does the :-jackfirth part make this default to jackfirth if DOCKER_USERNAME isn't defined?

Yep.

> I lean towards making this variable required with no default or not making it configurable at all. IMO, the fewer configuration knobs a shell script has the better.

I'll make it required and make a note of it in the README. I think this bit of configurability is necessary for contributors to be able to test the whole flow after they make changes (since you're the only one who can push to the `jackfirth/racket` repo).